x-pack/platform/plugins/shared/searchprofiler/public/application/components/profile_query_editor/profile_query_editor.tsx

```diff
@@ -139,41 +133,31 @@ export const ProfileQueryEditor = memo(() => {
                 />
               </EuiFormRow>
             </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiToolTip content={tooltipContent}>
+                <EuiButtonIcon
```

Comment:
can we add an aria-label to this button icon? otherwise will be hard for screen reader users to know what it is